payjoin/src/send/v2/session.rs

```diff
@@ -129,7 +129,7 @@ mod tests {
         };
 
         let v2_get_context = V2GetContext {
-            endpoint,
+            endpoint: endpoint.clone(),
```

Comment:
I dont see other changes that would warrant this clone? Do you remenber why we needed this?